src/renderer/components/+workloads-pods/pod-details-container.tsx

```diff
@@ -54,6 +55,15 @@ export class PodDetailsContainer extends React.Component<Props> {
           </span>
         </DrawerItem>
         }
+        {status &&
```

Comment:
I would change this to `{lastState &&`. So not displaying even the title if Last status is not found.

Reply:
That was a typo. Corrected in next commit